resources/cygnus-translators/README.md

```diff
@@ -1,7 +1,14 @@
-# Cygnus translators
+#<a name="top"></a>Cygnus translators
 
-## `cygnus-translator-0.1-to-0.2.sh`
+Content:
 
+* [`cygnus_translator_0.1_to_0.2.sh`](#section1)
+* [`cygnus_translator_0.1_to_0.3.sh`](#section2)
+* [`cygnus_translator_0.2_to_0.3.sh`](#section3)
+* [`cygnus_translator_0.9.0_to_0.10.0_hdfs.sh`](#section4)
```

Comment:
Re-fixed in https://github.com/telefonicaid/fiware-cygnus/commit/054578b9a04547026fbd86950a4e03a3c5012a42